tests/dualtor/test_ipinip.py

```diff
@@ -17,13 +17,24 @@
 from tests.common.dualtor.dual_tor_utils import rand_selected_interface
 from tests.common.dualtor.tunnel_traffic_utils import tunnel_traffic_monitor
 from tests.common.utilities import is_ipv4_address
-
+from tests.common.fixtures.ptfhost_utils import run_garp_service
+from tests.common.fixtures.ptfhost_utils import change_mac_addresses
 
 pytestmark = [
     pytest.mark.topology("t0")
 ]
 
 
+@pytest.fixture(scope="module", autouse=True)
+def mock_common_setup_teardown(
+    apply_mock_dual_tor_tables,
+    apply_mock_dual_tor_kernel_configs,
+    cleanup_mocked_configs,
+    request
+):
+    request.getfixturevalue("run_garp_service")
```

Comment:
Would this fail on T0? Since it still tries to get the MUX_CABLE table from config DB. I may refactor `garp_service` so that the configuration and deployment occur separately, that way `dual_tor_mock` can generate its own config.

Reply:
No, since I removed the `autouse` from `run_garp_service` so for testcases that wish to use `run_garp_service` it should explicitly call it. So the use of `request.getfixturevalue` to call `run_garp_service` here is to ensure that `run_garp_service`(session level) runs after those mocking fixtures(module level).

Reply:
I've tested it over T0 as described in the PR description.

Reply:
Oh I see now, thanks!